soh/soh/OTRGlobals.cpp

```diff
@@ -608,6 +611,17 @@ extern "C" void ResourceMgr_LoadFile(const char* resName) {
     OTRGlobals::Instance->context->GetResourceManager()->LoadResource(resName);
 }
 
+std::shared_ptr<Ship::Resource> ResourceMgr_LoadResource(const char* path) {
+    std::string Path = path;
+    if (ResourceMgr_IsGameMasterQuest()) {
+        size_t pos = 0;
+        if ((pos = Path.find("/nonmq/", 0)) != std::string::npos) {
+            Path.replace(pos, 7, "/mq/");
```

Comment:
This function intercepts most resource loads. Doesn't get called if MQ is not needed, and also no replace is attempted if nonmq isn't in the string at all.